test/e2e/scenarios/start/benefitType.js

```diff
@@ -25,3 +26,32 @@ Scenario('When I enter a non PIP benefit type, I am taken to the download form p
     I.seeInCurrentUrl(paths.identity.downloadAppointeeForm);
 
 });
+
+const benefit = new DataTable(['BenefitType', 'BenefitName']); //
+benefit.add(['Attendance Allowance', 'Attendance Allowance']); // adding records to a table
+benefit.add(['Bereavement Benefit', 'Bereavement Benefit']);
+benefit.add(['Carer’s Allowance', 'Carer’s Allowance']);
+benefit.add(['Disability Living Allowance (DLA)', 'Disability Living Allowance (DLA)']);
+benefit.add(['Employment and Support Allowance (ESA)', 'Employment and Support Allowance (ESA)']);
+benefit.add(['Incapacity Benefit', 'Incapacity Benefit']);
+benefit.add(['Industrial Injuries Disablement', 'Industrial Injuries Disablement']);
+benefit.add(['Jobseeker’s Allowance (JSA)', 'Jobseeker’s Allowance (JSA)']);
+benefit.add(['Maternity Allowance', 'Maternity Allowance']);
+benefit.add(['Severe Disablement Allowance', 'Severe Disablement Allowance']);
+benefit.add(['Social Fund', 'Social Fund']);
+benefit.add(['Universal Credit (UC)', 'Universal Credit (UC)']);
+
+Data(benefit).Scenario('When I enter differernt non PIP benefit types, I am taken to the download form page', (I, current) => {
+    I.enterBenefitTypeAndContinue(current.BenefitName);
+    I.see(current.BenefitName, {css: '.column-two-thirds>p'});
```

Comment:
Up until now we haven't used CSS selectors in our end-to-end tests. Is there a good reason for this?

Reply:
Agree with Paul. It's not an issue to use the css, but the question is: do you really need it? Why not just a simple 'I.see(current.BenefitName)'?